shared/templates/accounts_password/ansible.template

```diff
@@ -5,7 +5,31 @@
 # disruption = low
 - (xccdf-var var_password_pam_{{{ VARIABLE }}})
 
-- name: Ensure PAM variable {{{ VARIABLE }}} is set accordingly
+{{% if product == "ol8" %}}
+- name: {{{ rule_id }}} - Find pwquality.conf.d files
+  find:
+    paths: /etc/security/pwquality.conf.d/
+    patterns: "*.conf"
+  register: pwquality_conf_d_files
+
+- name: {{{ rule_id }}} - Ensure {{{ VARIABLE }}} is not set in pwquality.conf.d
+  lineinfile:
+    path: "{{ item.path }}"
+    regexp: '^\s*\b{{{ VARIABLE }}}\b.*'
+    state: absent
+  with_items: "{{ pwquality_conf_d_files.files }}"
+{{% endif %}}
+
+{{% if "ol" in product %}}
+{{{ ansible_remove_pam_module_option_configuration('/etc/pam.d/system-auth',
+                                  'password',
+                                  '',
+                                  'pam_pwquality.so',
+                                  VARIABLE)
+}}}
+{{% endif %}}
+
+- name: {{{ rule_id }}} - Ensure PAM variable {{{ VARIABLE }}} is set accordingly
```

Comment:
Please, to keep it aligned to other rules, use {{{ rule_title }}} instead of {{{ rule_id }}}